pages/data.md

```diff
@@ -1,35 +1,60 @@
+
+For this challenge, drought was characterized for each beetle sampling event using values of the Standardized Precipitation Evapotranspiration Index (SPEI) retrieved from the [GRIDMET Drought image collection](https://developers.google.com/earth-engine/datasets/catalog/GRIDMET_DROUGHT#description) (see [Abatzoglou (2011)](https://doi.org/10.1002/joc.3413).
+
+All data are licensed under <<get NEON preferred license infromation from Christine>>.
+
+## General Overview of Inputs and Expected Outputs
+
+**Model inputs** will be a flat metadata table (csv format) where each row is a metadata record for a specimen image. Multiple images will be associated with a sampling `event_id`, where a "sampleing event" corresponds with a location (NEON Site ID) and the date on which the beetle specimens were collected. Thus, multiple input records (multiple images of beetle specimens) will be used to predict a single SPEI value at a given site on a given date. Note that `event_id`s will be anonymized in the test dataset, but the metadata will include `site_id` and `collect_date` in the training data. 
+
+**What to predict:** The SPEI metrics represent cumulative drought conditions over a time window. For this challenge, participants will submit predictions for SPEI metrics representing three different time scales: 
+- `spei_30d` is drought conditions calcualted from data for the 30 day window preceding sample collection
+- `spei_1y` is the drought condition calculated from the year preceding sample collection
+- `spei_2y` is the drought condition calculated from the two years preceding sample collection
+
+**The submission file** will be a long-format flat table (csv file format) with a prediction for each `spei_30d`, `spei_1y`, and `spei_2y` for each `event_id`. Predictions will include a measure of uncertainty, so each prediction will include `mu` (mean) and `sigma` (standard devation). See the example submissoin below for more details. 
 
 ## Instructions to Download Training Data
 
+<<needs updating for beetle ml process, once we have it developed>>
+
 First, install the downloader in your virtual environment:
 ```bash
 pip install git+https://github.com/Imageomics/cautious-robot
 ```
-Then download [`butterfly_anomaly_train.csv`](https://github.com/Imageomics/HDR-anomaly-challenge/blob/main/files/butterfly_anomaly_train.csv) and run: 
+Then download <<link to training data csv>> and run: 
 ```bash
-cautious-robot -i <path/to/butterfly_anomaly_train.csv> -o <path/to/images> -s hybrid_stat -v md5
+cautious-robot -i <path/to/beetle_ml_train.csv> -o <path/to/images> -s hybrid_stat -v md5
 ```
 
-This will create subfolders `hybrid` and `non-hybrid` with images named by the `filename` column (`<CAMID>.jpg`). Remove the `-s hybrid_stat` if you want a flat directory.
+This will create subfolders ...
+<<are we going to provide a download of the images, or just a metadata file with image URIs?>>
```

Comment:
Also needs to be updated before merging? @sokole

Reply:
same here. I don't have this info. Lisa or someone will need to update this section.